isis/scripts/isisVarInit.py

```diff
@@ -42,6 +82,8 @@ def activate_text(shell: dict, env_vars: dict) -> str:
     if shell["activate_extra"] != "":
         lines.append(shell["activate_extra"])
 
+    lines.append(shell['path_update'])
```

Comment:
This is where things get dicey, and I'm worried about where this is going.  The default version of this program, and the activate scripts did not need to set the path, because the ISIS programs were simply in $CONDA_PREFIX/bin, and all of the path setting nonsense was taken care of by conda.  Putting this functionality into the activate script fundamentally circumvents all of the path wrangling that conda does, and I'm concerned that there is a high liklihood of downstream complications.  This should not be done in a script like this that we expect non-experts to use.

Explicitly doing this will place the $ISISROOT/bin into the PATH twice (once when conda sets up the path, and once when you run the activate script) when run in the current environment.  The fact that any "other" environment that you write these scripts into will get a path add, but not a path removal on conda deactivate seems like a real bad idea.  I appreciate that you can just "terminate the shell," but that requires that you know what you're doing.  I suspect that for less savvy users, this could result in a kind of memory leak situation, in which someone blithely uses the same shell and activates and deactivates one of these environments over and over again, with their PATH variable growing and growing and growing.